src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -30,11 +37,10 @@ public class OtpUser extends AbstractUser {
     public boolean isPhoneNumberVerified;
```

Comment:
For push notifications, we will need a new field (or several?) to hold the description or attributes of the device that will receive the notifications.

Reply:
Yes, we've been calling this a "device ID" though we know nothing of the format. I can put a String in there for now.

Reply:
The Mobile Application team's API keeps track of multiple device IDs, indexed by user (email address). So it seems we don't need a new field for this.